contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--2.3.0--2.4.0.sql

```diff
@@ -2505,6 +2505,55 @@ where has_schema_privilege(sch.schema_id, 'USAGE')
 and has_table_privilege(c.oid, 'SELECT,INSERT,UPDATE,DELETE,TRUNCATE,TRIGGER');
 GRANT SELECT ON sys.index_columns TO PUBLIC;
 
+/*
```

Comment:
why is this added to 2.3 to 2.4 in 3 x dev? what about 3.1 to 3.2?

Reply:
Added upgarde script from 3.1 to 3.2